Makefile

```diff
@@ -0,0 +1,29 @@
+NAME = heroku-cli
+
+VERSIONS = 7.5
```

Comment:
❔How should this look like when you want to support multiple versions?

Reply:
Further versions will be appended and separated by a space.

E.g.
```Makefile
VERSIONS = 7.5 7.6 7.7
```